cli/injector/main.go

```diff
@@ -329,6 +339,50 @@ func injectAndWait(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	if len(pulsingInjectors) > 0 {
+		isInjected := true
+		sleepDuration := pulseActiveDuration
+
+		for { // This loop will inject, wait, clean, wait until a signal is received
```

Comment:
This loop says it begins by injecting. However, haven't we _already_ injected by the time we get here? It looks like we start all the injectors, even the pulsing ones, at https://github.com/DataDog/chaos-controller/pull/473/files#diff-7a828d3d2d4e9739314e9b20ad4cde74c8b9fdcef19e29facad2ecb7f17e1b78R296

In that case, shouldn't we sleep for the dormant duration initially? I think the `if` condition correctly handles the clean/Inj lifecycle, we just start with the wrong duration, afaict